en/contributing/test.md

```diff
@@ -58,7 +58,7 @@ make posix gazebo
 
 Then run the tests as shown:
 ```
-make run_integration_tests
+make run_integration_tests 
```

Comment:
@julianoes How do we run integration tests for external examples?

Reply:
Yer that's currently broken :(.